guides/source/active_storage_overview.md

```diff
@@ -427,6 +427,16 @@ end
 
 You can configure specific variants per attachment by calling the `variant` method on yielded attachable object:
 
+If you are using **ruby-vips** (Rails 7+ default):
+```ruby
+class User < ApplicationRecord
+  has_one_attached :avatar do |attachable|
+    attachable.variant :thumb, resize_to_limit: [100, nil]
+  end
+end
+```
+
+If you are using **imagemajick**:
```

Comment:
Do you mean imagemagick?

Reply:
yes, probably - haven't been using Rails much for around 4 years 😢  and forgot to check the spelling